### ISSUE TITLE
Wrong ES2015 highlighting

### ISSUE THREAD

**Issue by diegotoral** [dracula/vim#23]
Yay! 

After installing `pangloss/vim-javascript` plugin I noticed that my `ES2015` code isn't been properly highlighted, some keywords (`export`, `import`, `from`) remain white (see screenshot). I've tried other themes and these keywords are been highlighted as expected.

<img width="272" alt="screenshot 2016-11-23 21 01 54" src="https://cloud.githubusercontent.com/assets/624418/20584233/76fad0c2-b1cf-11e6-8ad6-1ecd6b6f2164.png">

**Comment by petermbenjamin** [comment]
Same here.

![screen shot 2017-08-25 at 5 04 27 pm](https://user-images.githubusercontent.com/6811830/29736582-8593ffda-89b7-11e7-9556-510efc7884ce.png)

**Comment by zenorocha** [comment]
Can you take a look @heinst?

**Comment by zenorocha** [comment]
That's great @stephengfriend, what do you think of sending a PR with that change?

**Comment by benknoble** [comment]
/offtopic @stephengfriend for the whitespace characters see `:help 'list'` and `:help 'listchars'`

**Comment by stephengfriend** [comment]
@zenorocha I'll try to take some time over the weekend to exercise all of the javascript highlights and send up a PR.

@benknoble Yea, I was trying to implement vscode's version of _boundary_ whitespace characters. Where I want to show leading and trailing whitespace characters. Unfortunately, that takes some matching regex and such. Thanks for the direction, though.

**Comment by benknoble** [comment]
@stephengfriend `listchars` won't do leading, but it does have a trailing option.

**Comment by zenorocha** [comment]
That's great, thanks @stephengfriend!

**Comment by dsifford** [comment]
Just circling back to @diegotoral -- The primary reason for the botched syntax highlighting is due to the extremely outdated builtin javascript syntax that comes bundled with vim.

I'm not familiar with polyglot as @stephengfriend mentioned, so I can't speak to that directly (it does look pretty nice though, so I might check it out!), but I do know there are several more modern syntaxes available that you can use instead of the builtin one.

check www.vimawesome.com

@stephengfriend -- When I get some extra time, I'll take a look at adding the expanded polyglot groups to #62 :+1: 

Closing this issue because the fixable things that were broken in the builtin javascript syntax are going to be fixed in #62.

**Comment by stephengfriend** [comment]
@dsifford Sorry, I've been too busy lately and also stopped using [n]vim after discovering the limitations in it's syntax highlighting. If I give it another go, I'll try to add the javascript specific portions after #62 lands.

**Comment by dsifford** [comment]
No worries.

Curious: What do you mean when you say "discovering the limitations in it's syntax highlighting"? I wasn't aware of any specific differences other than nvim uses 256 colors by default and supports truecolor?
